tests/UI/campaigns/functional/BO/14_advancedParameters/05_import/02_importFile.js

```diff
@@ -0,0 +1,142 @@
+require('module-alias/register');
+
+const {expect} = require('chai');
+
+// Import utils
+const helper = require('@utils/helpers');
+const files = require('@utils/files');
+const loginCommon = require('@commonTests/loginBO');
+
+// Import pages
+const dashboardPage = require('@pages/BO/dashboard');
+const importPage = require('@pages/BO/advancedParameters/import');
+const customersPage = require('@pages/BO/customers');
+
+// Import test context
+const testContext = require('@utils/testContext');
+
+const baseContext = 'functional_BO_advancedParameters_import_importFile';
+
+// Variables
+let browserContext;
+let page;
+
+// Variable Used in the download / rename / upload test
+let sampleFilePath;
+const sampleFile = {
```

Comment:
Maybe you should create a demo file data for the import page.

Reply:
If this data is related to fixtures, it *must* be created in a demo data file.

Reply:
Done !